processing/src/main/java/io/druid/segment/DimensionHandlerUtils.java

```diff
@@ -291,4 +300,21 @@ public static Long getExactLongFromDecimalString(String decimalStr)
       return null;
     }
   }
+
+  public static Double convertObjectToDouble(Object valObj)
+  {
+    if (valObj == null) {
+      return 0.0d;
```

Comment:
Same as above, maybe return cached constant. Same for convertObjectToFloat(), convertObjectToLong().